charts/scalar-manager/templates/scalar-manager/configmap.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
As we discussed, we noticed that this `ConfigMap` overwrites some stored metadata (e.g., managed cluster or pause duration info) when we run the `helm upgrade` command. So, it would be better to remove this `ConfigMap` from the helm chart side and create it on the Scalar Manager API side.

Reply:
We address this issue on the Scalar Manager API side.
https://github.com/scalar-labs/scalar-manager-api/pull/90